test/testutils.py

```diff
@@ -47,6 +47,8 @@ def DictDiff(a, b):
   def Compare(x, y):
     if x == y:
       return True
+    if x in [None, 'nan'] and y in [None, 'nan']:
```

Comment:
Why do you need this?  And that says that `None` and `'nan'` are `==`.  Is that really what we want?

Reply:
Modified because `testTypeExamples` fails. Not ideal. Reverted.